whisper/whisperv5/api.go

```diff
@@ -96,12 +96,12 @@ func (api *PublicWhisperAPI) SetMinPoW(ctx context.Context, pow float64) (bool,
 
 // MarkTrustedPeer marks a peer trusted. , which will allow it to send historic (expired) messages.
 // Note: This function is not adding new nodes, the node needs to exists as a peer.
-func (api *PublicWhisperAPI) MarkTrustedPeer(ctx context.Context, enode string) (bool, error) {
-	n, err := discover.ParseNode(enode)
+func (api *PublicWhisperAPI) MarkTrustedPeer(ctx context.Context, url string) (bool, error) {
+	n, err := enode.ParseV4(url)
 	if err != nil {
 		return false, err
 	}
-	return true, api.w.AllowP2PMessagesFromPeer(n.ID[:])
+	return true, api.w.AllowP2PMessagesFromPeer(n.ID().Bytes())
```

Comment:
In v6, you write the same code as:
```go
id := n.ID()
err = api.w.SendP2PMessage(id[:], env)
```
I would appreciate if you used the same thing in both versions. This is so that if people (say, security auditors) want to compare v5 and v6, the diffs will show something that is as close as possible to only changes in the protocol. For that same reason, I would prefer if you picked `n.ID().Bytes()` over what is currently found in v6.